### DISCUSSION TITLE
Calendar?

### DISCUSSION THREAD

**Discussion by tomByrer** [anyproto/contributors#89] [Q&A] [answered]
I looked around, I could not find anything about a calendar.  For some, this will be a must-have to switch from Notion, amplenote, etc.
 Might be out of scope?

**Comment by isle9** [comment, marked as answer]
It's the top voted feature on the [forum](https://community.anytype.io/t/calendar-timeline-views/1576).

**Reply by tomByrer** [reply]
Thanks; I didn't want to create another login to just ask 1 question that I kinda guessed the answer for...
quite the thread you linked!
